WalletWasabi/Services/TransactionProcessor.cs

```diff
@@ -170,7 +162,7 @@ public bool Process(SmartTransaction tx)
 																																																																		   // If we did not have it.
 					if (Coins.TryAdd(newCoin))
 					{
-						TransactionCache.TryAdd(tx);
+						TransactionStore.TryAdd(tx);
```

Comment:
This is not necessary because the transaction is already known to be relevant so, it will be added at the end of this method.

Reply:
Good catch. Fixed.